project.clj

```diff
@@ -39,7 +39,8 @@
                                           :optimizations :simple
                                           :pretty-print false}}}}}}
   :cljsbuild {:builds {:client {:source-paths ["src/cljs/main"]
-                                :compiler {:main "broadfcui.main"}}}}
+                                :compiler {:main "broadfcui.main"
+                                           :output-to "target/compiled.js"}}}}
```

Comment:
About all I can do is confirm that this reintroduces `:output-to "target/compiled.js"` which certainly seems important.

I also noticed that commit removed `:output-dir` and `:asset-path`. Do those need to be revived as well?